test/Conversion/TorchToArith/basic.mlir

```diff
@@ -242,7 +242,7 @@ func.func @torch.aten.mul.int(%arg0: !torch.int, %arg1: !torch.int) -> !torch.in
 // CHECK-DAG:      %[[LHS_I64:.*]] = torch_c.to_i64 %[[LHS]]
 // CHECK-DAG:      %[[RHS_F64:.*]] = torch_c.to_f64 %[[RHS]]
 // CHECK:          %[[LHS_F64:.*]] = arith.sitofp %[[LHS_I64:.*]] : i64 to f64
```

Comment:
also here: `%[[LHS_I64:.*]]` => `%[[LHS_I64]]` and `MUL`, `OUT`

Reply:
Thanks. Pushed a new commit.